packages/shared/trade/src/transaction-confirmation/transaction-confirmation.tsx

```diff
@@ -178,10 +183,6 @@ const TermsOfService = styled('p')(
   margin-top: 1rem;
```

Comment:
In the future we can probably also switch this TermsOfService component to a Typography one, probably Body or something like that